tests/observability/metrics/conftest.py

```diff
@@ -243,14 +244,14 @@ def windows_vm_for_test_interface_name(windows_vm_for_test):
 
 
 @pytest.fixture(scope="class")
-def vm_with_cpu_spec(namespace, unprivileged_client):
+def vm_with_cpu_spec(namespace, unprivileged_client, nodes_cpu_architecture):
     name = "vm-resource-test"
     with VirtualMachineForTests(
         name=name,
         namespace=namespace.name,
         cpu_cores=TWO_CPU_CORES,
         cpu_sockets=TWO_CPU_SOCKETS,
-        cpu_threads=TWO_CPU_THREADS,
+        cpu_threads=ONE_CPU_THREAD if nodes_cpu_architecture == "s390x" else TWO_CPU_THREADS,
```

Comment:
`nodes_cpu_architecture == "s390x` - please extract to a session_scoped fixture - there are at least 2 more places in the code that check this